src/components/Edit/ChangeSummary.vue

```diff
@@ -43,25 +43,44 @@
       </div>
     </template>
 
+    <!-- Office Addresses -->
     <template v-if="officeAddressesChanged">
-      <v-divider class="mx-4" />
+      <v-divider class="mx-8" />
       <div id="address-summary-section" class="section-container">
         <OfficeAddresses :isSummaryView="true" />
       </div>
     </template>
+
+    <!-- Org Persons -->
+    <template v-if="hasOrgPersonChanged">
+      <v-divider class="mx-8" />
+      <div id="org-person-summary-section" class="section-container">
```

Comment:
Have you considered making this an actual `<section>` ? (Or article.) (And some of the other divs on this page.)

Reply:
I hadn't.
What is the difference/relevance, is it mostly semantics?
Real question :) I don't actually know the purpose!

Reply:
After a little research, will update. 
Looks like article is the more appropriate tag. Section does sound a little more root level, ie Header / Body / Footer
Obviously this brings into question some of our generic class names, but i won't get too hung up on it at the moment.

Reply:
Updated here

Reply:
A section can contain articles, and an article can contain sections. Theoretically, they are both at the same level. In practise, I try for consistency within the current app or page. In other words, if the main blocks are sections then I use articles for sub-blocks. Or vice-versa.

Reply:
I like it! Will try and follow this more closely in the future. 
Updated this file

Reply:
Looks good!